### ISSUE TITLE
Migration Issue - Missing isAuthenticated; and authenticate() 

### ISSUE THREAD

**Issue by KayHS** [auth0/lock#1297]
Hi,

I have an angular 1.6 application - i am trying to migrate an old version of auth0 v9 lock to v11.

property: auth.IsAuthenticated
function: auth.authenticate();

From what i can tell both the above property and function have been deprecated in the new version, but i was not able to find a replacement within the new lock v11 api or the migration.

Can someone help me understand what these functions did(i have an idea) and help me with determining what to replace them with.

thanks.

**Comment by luisrudge** [comment]
Hi, we have an extensive migration guide [here](https://auth0.com/docs/libraries/lock/v11/migration-guide) that you can take a look. If you have any questions, please reach out to our amazing support team at https://support.auth0.com

**Comment by KayHS** [comment]
@luisrudge Hey, i mentioned i looked at the migration guide and i cannot find it there. If you have seen it and could point it out that would be great.

**Comment by luisrudge** [comment]
https://auth0.com/docs/libraries/lock/v11/migration-angularjs-v9#add-the-angular-lock-library

You're probably using https://github.com/auth0/auth0-angular, which was deprecated. You need to use angular-lock

**Comment by KayHS** [comment]
@luisrudge Yes, that is the guide i have been following.

I removed auth0-angular in favour for...

 "auth0-lock": "^11.4.0",
"auth0.js": "^9.0.0",    
"angular-lock": "^3.0.0"

So i understand the auth object was changed to the lock object and both this property and function were removed. 

From my understanding 

-i use the AuthService isAuthentication method as a replacement for auth.isAuthenticated?

My main issue was more -

- What do i use as a replacement for the auth.authenticate function (not to be mistaken with auth.signin()).
